sources/packages/web/src/components/students/modals/ConfirmChangeApplication.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <modal-dialog-base title="Edit application" :showDialog="showDialog">
+    <template #content>
+      <v-checkbox
+        v-model="conditionsAccepted"
+        label="I acknowledge that by editing my application, I may be required to
+        resubmit previously approved exception requests. My institution may be
+        required to resubmit program information, and my parent or partner may be
+        required to resubmit supporting information."
+        hide-details="auto"
+      ></v-checkbox>
+      <banner
+        class="mb-4"
+        v-if="!isStudentEndDateWithinDeadline"
```

Comment:
We must show the banner when the study end date is within six weeks. i.e. `v-if="isStudentEndDateWithinDeadline"`